### PR TITLE
Carbon monitor

### PR DESCRIPTION
**Pull request by lgloege**
This PR adds processings and import script for Carbon Monitor and makes a new alembic migration

### CODE REVIEW

File: global-api/importer/carbon_monitor/process.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            "Korea": "Korea, the Republic of",
+            "Turkey": "Türkiye",
+            "Russia": "Russian Federation",
+        }
+    }
+
+    df = (
+        pd.read_csv(INPUT_FILE, parse_dates=["date"])
+        # 0. filter out rows with 0 emissions
+        .loc[lambda x: x['value (KtCO2 per day)']>0]
```
Comment (lemilonkh) [inline global-api/importer/carbon_monitor/process.py:53]: Why are you filtering out the 0 values? Wouldn't these still be valid and good to have present, so the user doesn't have to research them themselves?

Or are these invalid

**Review by lemilonkh — APPROVED** [review]
Looks great 👍
I think it would be helpful to add an extraction script that downloads the file from the given URL, so we can integrate this data source into Airflow in the future and process it automatically.

This can be handled in a future PR though.

**Comment by evanp** [comment]
@mfonsecaOEF @lgloege I don't think this `CarbonMonitor` table is necessary. We have a `citywide_emissions` table. Can you use that instead? Mau, can you help Luke with this?

**Comment by evanp** [comment]
So, it looks like there are outstanding issues here. @lgloege can you resolve?

File: global-api/importer/carbon_monitor/process.py (hunk 1 above)
Comment (evanp) [reply global-api/importer/carbon_monitor/process.py]: I discussed with @lgloege and our consensus is that it probably makes sense to be conservative in what we import. Zeroes may be measured values, but they may also signal a lack of information.

**Review by evanp — COMMENTED** [review]
Seems to be well-reviewed.

**Pull request merged**